packages/gatsby/src/utils/worker/__tests__/queries.ts

```diff
@@ -180,9 +180,14 @@ describeWhenLMDB(`worker (queries)`, () => {
     }
   })
 
+  // This was the original implementation of state syncing between a worker and the main process.
+  // We switched to "replaying actions" as a mechanism for state syncing.
+  // But we can get back to state saving / merging if "replaying actions" proves to be too expensive
+  // TODO: delete or re-activate depending on results yielded by "replaying actions" approach.
```

Comment:
We could add that this then can be removed since we test the `loadPartialStateFromDisk` logic in `share-state` test. Because at first I though "mhh, maybe keep this to test that other logic" but yeah, we already test this somewhere else

Reply:
Just to clarify - do you propose to expand this comment? Maybe add a suggestion via GitHub UI?

Reply:
```suggestion
  // TODO: delete or re-activate depending on results yielded by "replaying actions" approach.
  // The logic for `loadPartialStateFromDisk` itself is tested in `share-state` tests
```

Reply:
Done!